lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -336,13 +336,21 @@ namespace Js
     bool
     FunctionBody::SkipAutoProfileForCoroutine() const
     {
-        return this->IsCoroutine() && CONFIG_ISENABLED(Js::JitES6GeneratorsFlag);
+#if defined(_ARM64_) || defined(_ARM_)
+        return false;
```

Comment:
Is this meant to be a `this->IsCoroutine();` to disable jilting generators for ARM rather than just always returning false - I think there's a couple of non-generators that will call this.

Reply:
No this is intended because we don't want to skip auto profiling at all if jitting coroutines is not enabled.